vllm/model_executor/models/llama.py

```diff
@@ -520,6 +533,13 @@ def __init__(self,
         self.make_empty_intermediate_tensors = (
             self.model.make_empty_intermediate_tensors)
 
+    def set_aux_hidden_state_layers(self, layers: list[int]) -> None:
+        self.model.aux_hidden_state_layers = layers
+
+    def get_eagle3_aux_hidden_state_layers(self) -> list[int]:
+        num_layers = len(self.model.layers)
+        return [2, num_layers // 2, num_layers - 3]
```

Comment:
Is this same for all sizes of the model?

Reply:
Also, is this same for all models? Or, is it only the case for Llama?

Reply:
This is unclear, as the EAGLE reference code seems to only support Llama models. The relevant LOC is [here](https://github.com/SafeAILab/EAGLE/blob/91ae5f2ffa44a6e2cf50b7a1d19d899c7bbd5817/eagle/model/modeling_llama_kv.py#L1138). It is therefore the same for all Llama models.

I think it's best to keep this as a per-model-class parameter until we have reason to believe otherwise.